packages/core/src/components/Sheet/index.tsx

```diff
@@ -377,6 +393,14 @@ const InternalSheet = withStyles(
       paddingTop: 0,
       flex: 1,
     },
+
+    contentCompact: {
+      padding: 1 * unit,
+    },
+
+    headerShadow: {
+      boxShadow: '0px 2px 4px rgba(0, 0, 0, 0.1)',
```

Comment:
this looks real close/similar enough to `ui.boxShadow`. please use that for consistency